tests/pytests/unit/modules/test_win_lgpo_reg.py

```diff
@@ -47,7 +47,24 @@ def empty_reg_pol():
 
 
 @pytest.fixture
-def reg_pol():
+def empty_reg_pol_user():
+    class_info = salt.utils.win_lgpo_reg.CLASS_INFO
+    reg_pol_file = pathlib.Path(class_info["User"]["policy_path"])
+    if not reg_pol_file.parent.exists():
+        reg_pol_file.parent.mkdir(parents=True)
+    with salt.utils.files.fopen(str(reg_pol_file), "wb") as f:
+        f.write(salt.utils.win_lgpo_reg.REG_POL_HEADER.encode("utf-16-le"))
+    salt.utils.win_reg.delete_key_recursive(hive="HKLM", key="SOFTWARE\\MyKey1")
```

Comment:
Shouldn't this be HKCU considering function name suggests its the user hive.

Maybe I'm missing something as I'm not familiar with the codebase, just making sure its not a typo!